.zuul.yaml

```diff
@@ -2,20 +2,22 @@
     name: alerta-build-image
     parent: build-docker-image
     description: Build alerta container image
-    timeout: 2700  # 45 minutes
+    timeout: 2700 # 45 minutes
```

Comment:
There are sometimes funny linters complaining when line comment is not separated with 2 spaces, should be fine though